test/tests/parallel/set3/scripts/postconf-helper.bats

```diff
@@ -0,0 +1,61 @@
+load "${REPOSITORY_ROOT}/test/helper/common"
+load "${REPOSITORY_ROOT}/test/helper/setup"
+
+BATS_TEST_NAME_PREFIX='[Scripts] (helper functions) (postfix.sh) (_add_to_or_update_postfix_main) '
+CONTAINER_NAME='dms-test-main.cf-changes'
+
+# Various tests for the helper function _add_to_or_update_postfix_main()
+
+function setup_file() {
+  _init_with_defaults
+  _common_container_setup
+
+  # remove 'relayhost' from main.cf
+  _run_in_container postconf -X relayhost
+  assert_success
+
+  export CONFIG=/etc/postfix/main.cf
```

Comment:
Not necessary if you use extract out the common pattern in test cases to a method that can just reference this path directly.

```suggestion
```